app/models/contributor.rb

```diff
@@ -0,0 +1,4 @@
+class Contributor < ActiveRecord::Base
+  has_many :suggested_councillors, through: :councillor_contributions
```

Comment:
`suggested_councillor_spec.rb`and`user_contributes_new_councillor_spec.rb`pass without this association. Currently we don't have the `councillor_contribution_controller_spec.rb`. so if we get to the stage to have the test, we may need the association? but for now I deleted it.